src/pcapi/settings.py

```diff
@@ -69,7 +69,7 @@
 
 
 # SENTRY
-SENTRY_DSN = "https://0470142cf8d44893be88ecded2a14e42@logs.passculture.app/5"
+SENTRY_DSN = os.environ.get("SENTRY_SAMPLE_RATE", "https://0470142cf8d44893be88ecded2a14e42@logs.passculture.app/5")
```

Comment:
Il y a une typo je pense :)
```suggestion
SENTRY_DSN = os.environ.get("SENTRY_DSN", "https://0470142cf8d44893be88ecded2a14e42@logs.passculture.app/5")
```